docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -0,0 +1,200 @@
+
+### Step 1: Defining the Ops
+
+In Dagster, the minimum unit of computation is an op. This directly corresponds to an operator in Airflow. Here, we map the Airflow operators `t1`, `t2`, and `t3` to their respective Dagster op's.
+
+```python file=/integrations/airflow/tutorial_rewrite_dagster.py startafter=start_ops endbefore=end_ops
+@op
+def print_date(context) -> datetime:
+    ds = datetime.now()
+    context.log.info(ds)
+    return ds
+
+
+@op(retry_policy=RetryPolicy(max_retries=3), ins={"start": In(Nothing)})
+def sleep():
+    time.sleep(5)
+
+
+@op
+def templated(context, ds: datetime):
+    for _i in range(5):
+        context.log.info(ds)
+        context.log.info(ds - timedelta(days=7))
+```
+
+this yields us the following graph of computations
+
+<center>
+  <Image
+    alt="Screenshot of the dagster UI, showing the newly created graph of tutorial Ops"
+    src="/images/integrations/airflow/airflow_tutorial_rewrite_ops.png"
+    width={2200}
+    height={1300}
+  />
+</center>
+
+##### Op Level Retries
+
+In the tutorial DAG, the `t2` operator allowed for 3 retries. To configure the same behavior in Dagster, you can use [op level retry policies](/concepts/ops-jobs-graphs/op-retries)
+
+### Step 2: Defining the Job
+
+In Dagster, the computations defined in `op`s are composed in `job`s, which define the sequence and dependency structure of the computations you want to execute. This directly corresponds to a DAG in Airflow. Here, we compose the `op`s `print_date`, `sleep` and `templated` to match the dependency structure defined by the Airflow operators `t1`, `t2`, and `t3`.
+
+```python file=/integrations/airflow/tutorial_rewrite_dagster.py startafter=start_job endbefore=end_job
+@job(tags={"dagster/max_retries": 1, "dag_name": "example"})
+def tutorial_job():
+    ds = print_date()
+    sleep(ds)
+    templated(ds)
+```
+
+##### Job Level Retries
+
+Job level retries are managed by the [run launcher](/deployment/run-retries) you will need to enable support for them in your dagster.yaml, once you do you can define the retry count on the job.
+
+### Step 3: Defining the schedule
+
+In Dagster, schedules can be defined for jobs, which determine the cadence at which a job is triggered to be executed. Below we define a schedule that will run the `tutorial_job` daily:
+
+```python file=/integrations/airflow/tutorial_rewrite_dagster.py startafter=start_schedule endbefore=end_schedule
+schedule = ScheduleDefinition(job=tutorial_job, cron_schedule="@daily")
+```
+
+### Step 4: Running locally
+
+In order to run our newly defined Dagster job we'll need to define a Dagster [repository](/concepts/repositories-workspaces/repositories). A Dagster repository is a logical grouping of jobs and schedules that provides an entrypoint for viewing and executing them.
```

Comment:
> To run our newly defined Dagster job, we'll need to define a Dagster [repository](/concepts/repositories-workspaces/repositories). A Dagster repository is a logical grouping of jobs and schedules that provides an entry point for viewing and executing them.

Editing for clarity. With the shift to `Definitions`, we should update this section if everything will work with `Definitions`.